ui.frontend/src/view/FormFieldBase.js

```diff
@@ -499,7 +540,7 @@ class FormFieldBase extends FormField {
                 if (typeof this[fn] === "function") {
                     this[fn](change.currentValue, state);
                 } else {
-                    console.error(`changes to ${change.propertyName} are not supported. Please raise an issue`)
+                    console.warn(`changes to ${change.propertyName} are not supported. Please raise an issue`)
```

Comment:
I believe this should be error only

Reply:
Error means, something has broke, in this case, the idea is the following,
```
When a property change occurs in the model but there is no corresponding update method in the view, logging an warning will provide visibility into the issue. It can help developers/partners quickly identify missing or incomplete implementations and prompt us to address them.
```